### PR TITLE
chore: dependabot updates

### PR DESCRIPTION
**Pull request by jonathannorris**

### CODE REVIEW

**Comment by jonathannorris** [comment]
* **#26** <a href="https://app.graphite.dev/github/pr/DevCycleHQ/feature-flag-code-usage-action/26?utm_source=stack-comment-icon" target="_blank">Graphite</a> 👈
* `main`

This stack of pull requests is managed by Graphite. <a href="https://stacking.dev/?utm_source=stack-comment">Learn more about stacking.</a>
<h2></h2>

 Join @jonathannorris and the rest of your teammates on <a href="https://graphite.dev?utm-source=stack-comment">Graphite <b>Graphite</b></a>

File: dist/index.js
Comment (suthar26) [inline dist/index.js:1]: should we have the dist folder here? or is it specific to the code-usages and that is why we need it?

Comment (jonathannorris) [reply dist/index.js:1]: I think we need it, there is a `dist/action.js` file. it seems like husky is generating this file on commit...

Comment (jonathannorris) [reply dist/index.js:1]: yea I think these are just the axios changes getting added to it.

**Review by suthar26 — APPROVED** [review]

**Pull request merged**